dagmc_stats/DagmcQuery.py

```diff
@@ -1,11 +1,21 @@
 from pymoab.rng import Range
 from pymoab import core, types
+import pandas as pd
+import numpy as np
 import warnings
 
 class DagmcQuery:
     def __init__(self, dagmc_file, meshset=None):
         self.dagmc_file = dagmc_file
-        self.meshset = meshset
+        if meshset is None:
+            self.meshset = self.dagmc_file.root_set
```

Comment:
We should be sure to document in the docstring (we need docstring) that the de facto default is the root set.